test/README.md

```diff
@@ -1,7 +1,48 @@
 # Service Testing
 
-Testing gRPC services as black boxes.
+There are two ways to test the service APIs on this demo:
+
+1. Using black box testing, calling gRPC services
+and validating its direct response
+2. Using Trace-based tests, calling services
```

Comment:
Should this also include `gRPC`?

Reply:
No, because tests to the email service and frontend service use HTTP calls directly.